Samples.Swift/Samples.Swift/ViewControllers/CheckoutViewController/CheckoutViewController.swift

```diff
@@ -0,0 +1,478 @@
+//
+//  CheckoutViewController.swift
+//  paypalcheckout-iOS-swift-sample
+//
+//  Created by Jax DesMarais-Leder on 3/10/21.
+//  Copyright © 2021 PayPal. All rights reserved.
+//
+
+import UIKit
+import PayPalCheckout
+
+class CheckoutViewController: UIViewController, AddItemViewControllerDelegate {
+
+  let tableView = UITableView()
+  var checkoutButton = UIButton()
+  let paypalButton = PayPalButton(size: .expanded)
+  var checkoutFlowOption = UISegmentedControl()
```

Comment:
In other files we use lazily instantiated vars for our UI elements - is there a reason to not do that here? I think i'd prefer that to setting these once and then re-setting them in the `setup` function.